### PR TITLE
Add game, gui, and window focus functions

### PR DESCRIPTION
**Pull request by Foundit3923**
**This pull requests makes the following changes:**

- *Fixes issue #6
- *Add's pywinctl to dependencies in .toml
- *Add's window management functions in .core.utils
- *Game window can be focused with `set_nms_window_focus()`
- *Gui window can be focused with `set_gui_window_focus()`
- *Additional functions enable users to manage other windows
- *All functionality available through `import pymhf.core.utils`

### CODE REVIEW

File: pymhf/core/utils.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
 from ctypes import windll, create_unicode_buffer, byref, c_ulong
 from configparser import ConfigParser
 from typing import Optional
+import pywinctl as pwc
+from pywinctl import Window
```
Comment (monkeyman192) [inline pymhf/core/utils.py:5]: this import isn't used

File: pymhf/core/utils.py [hunk 2]
```diff
@@ -9,6 +13,30 @@
...
+def is_window_active(windowName):
+    return windowName == pwc.getActiveWindowTitle()
+
+
+def activate_window(windowName):
+    if is_window_launched(windowName):
+        get_window(windowName).activate()
+
+
+def set_nms_window_focus():
```
Comment (monkeyman192) [inline pymhf/core/utils.py:33]: this shouldn't be here. `pyMHF` is generic and not for NMS specifically. Any functions and methods need to be able to applied to any application, not just NMS

File: pymhf/core/utils.py (hunk 1 above)
Comment (Foundit3923) [reply pymhf/core/utils.py:5]: I imported the Window class here to avoid an implicit dependency when initializing a class object.

Comment (Foundit3923) [reply pymhf/core/utils.py]: fixed

File: pymhf/core/utils.py (hunk 2 above)
Comment (Foundit3923) [reply pymhf/core/utils.py]: fixed

**Pull request closed without merging**